applications/StructuralMechanicsApplication/custom_response_functions/adjoint_conditions/adjoint_semi_analytic_point_load_condition.cpp

```diff
@@ -123,7 +123,7 @@ namespace Kratos
         const SizeType number_of_nodes = this->GetGeometry().size();
         const SizeType dimension =  this->GetGeometry().WorkingSpaceDimension();
         const SizeType num_dofs = number_of_nodes * dimension;
-        rOutput = ZeroMatrix(number_of_nodes, num_dofs);
+        rOutput = ZeroMatrix(0, num_dofs);
```

Comment:
Does this make sense, I mean is it possible to have a 0xn matrix?

Reply:
size2() was used in a check later. This was my suggestion, but I see now it is confusing. Can we fix the check instead so that we can assign ZeroMatrix(0,0) here?

Reply:
I am not against this, i was just wondering if it is possible. But as i understand, the matrix object stores size1 and size2 as members so even though a matrix where one of those is 0 technically can not have any values, one can still retrieve the other.